pycontrails/core/met.py

```diff
@@ -2248,6 +2247,26 @@ def shift_longitude(data: XArrayType) -> XArrayType:
     ).sortby("longitude", ascending=True)
 
 
+def shift_longitude_reverse(data: XArrayType) -> XArrayType:
+    """Shift longitude values from [-180, 180) to [0, 360) domain.
```

Comment:
I think both in this docstring and the one above, we can remove this part:

> Shift longitude values ~~from [-180, 180)~~ to [0, 360) domain.

The implementation works for any starting domain.

Reply:
Removed